css/media.css

```diff
@@ -0,0 +1,1201 @@
+@media only screen and (max-width: 1024px) {
+	.header-content ,
+	.poster-content ,
+	.product-content {
+		max-width: 1024px;
+	}
+
+	.header-container ,
+	.poster-container ,
+	.product-container ,
+	.product-items {
+		max-width: 930px;
+	}
+
+	.poster-container{
+		height: 252px;
+	}
+		
+	.product {
+		margin-bottom: 75px;
+	}
+		
+	.product-tile {
+		max-width: 208px;
+	}
+
+	.product-tile__image {
+		max-width: 169px;
+	}
+
+	.product-tile:not(:last-child):after {
+		right: -17px;
+	}
+
+/*info start*/
+	.info {
+		max-width: 1024px;
+		margin-bottom: 93px;
+	}
+
+	.info-content {
+		max-width: 930px;
+		height: 442px;
+	}
+	
+	.info-container {
+		max-width: 854px;
+		margin: 0 38px;
+	}
+
+	.info__item {
+    width: 55vw;
+    height: 35vh;
+		margin-right: 46px;
+	}
+
+	.info-wrap {
+		max-width: 248px;
+		margin-bottom: 20px;
+	}
+
+	.info-wrap__title {
+		margin-bottom: 8px;
+	}
+
+	.info-wrap__desc {
+		font-size: 13px;
+		line-height: 16px;
+		margin-bottom: 14px;
+	}
+/*info end*/
+
+	/*stories start*/
+	.storie {
+		max-width: 1024px;
+		margin: 0 auto 85px auto;
+	}
+
+	.storie-content {
+		max-width: 930px;
+	}
+
+	.storie-container {
+		max-height: 370px;
+	}
+
+	.storie-link {
+		max-width: 100%;
+		margin: 0 0 37px 0;
+	}
+
+	.storie-wrap__desc {
+		font-size: 13px;
+		line-height: 16px;
+		margin: 0 auto 23px auto;
+	}
+	/*stories end*/
+
+	/*gallery start*/
+	.gallery {
+		max-width: 1024px;
+	}
+
+	.gallery-content {
+		max-width: 930px;
+	}
+
+	.gallery-wrap__subtitle {
+		margin: 0;
+	}
+
+	.gallery-items {
+		grid-template-columns: repeat(7, 1fr);
+	}
+	/*gallery end*/
+
+	/*footer start*/
+	.footer-content {
+		max-width: 1024px;
+	}
+
+	.footer-container {
+		max-width: 930px;
+		padding-top: 70px;
+		margin-bottom: 28px;
+	}
+
+	.singup-list__btn {
+		margin-top: 4px;
+		margin-bottom: 31px;
+	}
+
+	.footer-rights {
+		padding-bottom: 20px;
+	}
+
+	.footer-logo {
+		margin-right: 15px;
+	}
+
+	.footer-logo__subtitle {
+		font-size: 8px;
+	}
+	/*footer end*/
+}
+
+@media only screen and (max-width: 768px) {
+	
+	html ,
+	body {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Project_Axion/pullRequest?prid=5457134)